runtime/Makefile

```diff
@@ -73,10 +73,9 @@ BINARY_NAME=sledgert
 # To log, run `call http_total_log()` while in GDB
 # CFLAGS += -DLOG_TOTAL_REQS_RESPS
 
-# This flag logs the total number of sandboxes in the various states
+# This flag tracks the total number of sandboxes in the various states
 # It is useful to debug if sandboxes are "getting caught" in a particular state
-# To log, run `call runtime_log_sandbox_states()` while in GDB
-# CFLAGS += -DLOG_SANDBOX_COUNT
+# CFLAGS += -DSANDBOX_STATE_TOTALS
```

Comment:
I changed this flag to enable or disable per state atomic counts. I eliminated the bespoke function to print this as it's trivial to pretty print the structure directly in gdb. IMO, I think our debugging code become substantially more maintainable when we focus on accumulating debugging state that we can inspect inside gdb. This sort of presumes that all contributors get comfortable with gdb.